tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -137,6 +138,12 @@ def init(self):
 
         self.ansible_host.copy(src=ANSIBLE_LOGANALYZER_MODULE, dest=os.path.join(self.dut_run_dir, "loganalyzer.py"))
 
+        return self._setup_marker()
+
+    def _setup_marker(self):
+        """
+        Adds the marker to the syslog
+        """
         start_marker = ".".join((self.marker_prefix, time.strftime("%Y-%m-%d-%H:%M:%S", time.gmtime())))
         cmd = "python {run_dir}/loganalyzer.py --action init --run_id {start_marker}".format(run_dir=self.dut_run_dir, start_marker=start_marker)
```

Comment:
If the start marker has whitespace in it, loganalyzer.py may fail to parse the `--run-id` argument. Maybe we should enforce that the start_marker does not have whitespace? Or convert any whitespace in it to char like `_`.

Reply:
fixed

Reply:
Thanks! With your fix, #1747 would be unnecessary. I am going to close that one.